src/highlighted-card.tsx

```diff
@@ -119,6 +121,12 @@ const Content = React.forwardRef<HTMLDivElement, Props>((props, ref) => {
             </div>
             {imageUrl && (
                 <div
+                    {...(imageAlt
```

Comment:
Maybe we could update this in order to use a Mistica Image component instead of a div

Reply:
I woudn't touch this component too much. The idea is to deprecate/remove it mid-term